testData/solutions/linux.json

```diff
@@ -1430,15 +1430,61 @@
             "configuration": {
                 "type": "gpii.xrandr",
                 "capabilities": [
-                    "display.screenEnhacements",
+                    "display.screenEnhancement",
                     "applications.org\\.freedesktop\\.xrandr.id"
-                ]
-           }
+                ],
+                "capabilitiesTransformations": {
+                    "screen-resolution": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "inputPath": "http://registry\\.gpii\\.net/common/screenResolution",
+                            "defaultInputValue": "normal",
+                            "options": {
+                                "high": {
+                                    "outputValue": {
+                                        "literalValue": {
+                                            "width": 1366,
+                                            "height": 768
+                                        }
+                                    }
+                                },
+                                "normal": {
+                                    "outputValue": {
+                                        "literalValue": {
+                                            "width": 1366,
+                                            "height": 768
```

Comment:
yup, agreed @amb26 - and this is true for both this one and the windows one that @SteveALee is working on.

As @javihernandez mentioned with a reference to the JIRA (https://issues.gpii.net/browse/GPII-1186?focusedCommentId=14506&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-14506) this isn't ready for master yet, as we need to detect available resolutions, but it can go into review4, where we can hardcode the resolutions for the demo machine